examples/g_code_processor_dem.py

```diff
@@ -0,0 +1,82 @@
+import os
+import pybullet as p
+import pybullet_data
+import pybullet_industrial as pi
+import numpy as np
+
+
+def actuate_gripper(gripper: pi.Gripper, val: int):
+    return gripper.actuate(val)
+
+
+def couple_endeffector(gripper: pi.Gripper, robot: pi.RobotBase, link: chr):
+    return gripper.couple(robot, link)
+
+
+def decouple_endeffector(gripper: pi.Gripper):
+    return gripper.decouple()
+
+
+if __name__ == "__main__":
+
+    dirname = os.path.dirname(__file__)
+    urdf_file1 = os.path.join(dirname,
+                              'robot_descriptions', 'comau_nj290_robot.urdf')
+    urdf_file2 = os.path.join(dirname,
+                              'robot_descriptions', 'gripper_cad.urdf')
+
+    pysics_client = p.connect(p.GUI, options='--background_color_red=1 ' +
+                                             '--background_color_green=1 ' +
+                                             '--background_color_blue=1')
+    p.setPhysicsEngineParameter(numSolverIterations=10000)
+    p.configureDebugVisualizer(p.COV_ENABLE_GUI, 0)
+    p.setPhysicsEngineParameter(numSolverIterations=10000)
+    p.setAdditionalSearchPath(pybullet_data.getDataPath())
```

Comment:
This line is called twice for some reason

Reply:
The line was removed